controllers/barbican_controller.go

```diff
@@ -555,22 +538,32 @@ func (r *BarbicanReconciler) reconcileDelete(ctx context.Context, instance *barb
 
 // fields to index to reconcile when change
 const (
-	passwordSecretField     = ".spec.secret"
-	caBundleSecretNameField = ".spec.tls.caBundleSecretName"
-	tlsAPIInternalField     = ".spec.tls.api.internal.secretName"
-	tlsAPIPublicField       = ".spec.tls.api.public.secretName"
+	passwordSecretField      = ".spec.secret"
+	caBundleSecretNameField  = ".spec.tls.caBundleSecretName"
+	tlsAPIInternalField      = ".spec.tls.api.internal.secretName"
+	tlsAPIPublicField        = ".spec.tls.api.public.secretName"
+	p11LoginSecretField      = ".spec.pkcs11.loginSecret"
+	p11ClientDataSecretField = ".spec.pkcs11.clientDataSecret"
```

Comment:
Consider using `pkcs11XXX` instead of `p11XXX` for consistency and clarity.

Reply:
Done